docs/local-dependencies.md

```diff
@@ -0,0 +1,45 @@
+# Local dependencies
+
+## Python and virtual environment
+
+It is recommended to work within a virtualenv to ensure proper dependencies isolation.
+If you're not familiar with that concept, read [Python Virtual Environments - a Primer][].
+
+Alright, now you can [install virtualenv][install-virtualenv] and then type these commands knowing what you are doing:
+
+```shell
+$ virtualenv --python=python2.7 venv
+$ source venv/bin/activate
+$ pip install -r udata/requirements/develop.pip
+$ pip install -e udata/
+```
+
+
+## NodeJS and modules
+
+You need NodeJS 4.2. If it's not already installed or you have a different version,
+you should consider [installing NVM][nvm-install] which uses the existing `.nvmrc`.
+
+```shell
+$ nvm install
+$ nvm use
+```
+
+Then install JavaScript dependencies:
+
+```shell
+$ npm install
+```
+
+Once it's done, you should be able to run the build command for JS and CSS:
+
+```shell
+$ inv assets_build
```

Comment:
As I documented in launching-tasks, we already have two launchers and it's already too much, I prefer to avoid npm references in that context.